### ISSUE TITLE
Error installing Mac

### ISSUE THREAD

**Issue by Andrew-Sor** [balopez83/macOS_On_Hyper-V#26]
When I start installing the Mac, "Loading Installation Information" appears, and then this error appears:
![Снимок экрана 2024-03-30 172427](https://github.com/balopez83/macOS_On_Hyper-V/assets/133566800/76869213-616f-4a91-bea5-859e01d583a3)
How to fix it?

**Comment by balopez83** [comment]
@Andrew-Sor Looks like you don't have network set up properly in hyper-v

**Comment by balopez83** [comment]
See the instructions to make sure you set up the vm correctly to pass internet to the vm

**Comment by Andrew-Sor** [comment]
No. Before that, I installed another recovery and everything was fine there. But the process was very long and it did not install the Mac I wanted.
I created a new virtual machine with the same settings and everything is fine there.
Thank you for your response.
I have a new question... What should I do about shutting down the virtual machine? The next time you start it, will the system start and everything will be fine?
